Activities/Cryptography/UiPath.Cryptography.Activities/Models/CryptographyLocalItem.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using UiPath.Platform.ResourceHandling;
+
+namespace UiPath.Cryptography.Activities.Models
+{
+    /// <summary>
+    /// Used to convert cryptography files as IResources.
+    /// </summary>
+    public class CryptographyLocalItem : ILocalResource
+    {
+        /// <summary>
+        /// Create the ILocalResource implementation
+        /// </summary>
+        /// <param name="byteArray"></param>
+        /// <param name="fullName"></param>
+        internal CryptographyLocalItem(byte[] byteArray, string fullName)
```

Comment:
Hm, I think you need to decrypt immediately in the activity anyway if you want to output to a file as well (not just to a resource).

I think we need to clarify with the PM what the desired behavior is if you have set both the IResource output and the file output. Based on that discussion, this may be OK going forward and we can introduce a new "lazy" constructor when the need arises.